runtime/JavaScript/spec/imports/NodeCommonJSImportSpec.cjs

```diff
@@ -0,0 +1,31 @@
+"use strict";
+var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
```

Comment:
I could use an explanation of the code in this file. Why can't we just write "require('antlr4')" ?

Reply:
Right, I just transpiled it with ts, but you are right require will do the job. Changed :+1: